Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -1245,6 +1245,7 @@ class WebPage final : public API::ObjectImpl<API::Object::Type::BundlePage>, pub
     void setIsolatedTree(Ref<WebCore::AXIsolatedTree>&&);
 #endif
     NSObject *accessibilityObjectForMainFramePlugin();
+    bool fallbackToWebContentAXObjectForMainFramePlugin() const;
```

Comment:
I recognize this method name is already long, so will leave this as a judgement call for you to make. But in line with [code style guidelines](https://webkit.org/code-style-guidelines/#names-bool), maybe this should have a `should` prefix on the method name?

Reply:
Hmm... I started with `should` but dropped it for brevity, but maybe there's more virtue in following the guide.

Reply:
Yeah this is a tough one, either option is fine I think